README.md

```diff
@@ -1,13 +1,16 @@
-# DEVTOOLS-X
 
-`devutils` is macOS-only, and `devtoys` is Windows-only.  
-So here's DevTools-X -- an x-platform collection of dev-tools that is lighter, safer, and full of feature rich (currenltly 32 modules and growing) development utilities.
+<h1>
+    <img src="assets/logo.min.png" alt="Logo" height="20">
+    <span>DEVTOOLS-X</span>
+</h1>
+
+**DevTools-X** is a cross-platform collection of development utilities, designed to be lightweight, secure, and feature-rich. Unlike `devutils` (macOS-only) and `devtoys` (Windows-only), DevTools-X works seamlessly across multiple operating systems. With over 40 modules and counting, it provides a powerful toolkit for developers everywhere.
```

Comment:
I think that this shifts the focus more directly onto DevTools-X, while still acknowledging the inspiration from other tools like `devutils` and `devtoys`.

Reply:
This is perfect, feel free to remove the text below it: "Note: Application is under development, expect some bugs." :P it's stable IMO we can remove it